### DISCUSSION TITLE
Need help regarding Make File

### DISCUSSION THREAD

**Discussion by shikhar229169** [Cyfrin/foundry-full-course-cu#212] [Q&A] [answered]
I tried to install Make through the command
```
sudo apt install make
```

But it throws the error
```
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Note, selecting 'make-guile' instead of 'make'
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 guile-3.0-libs : Depends: libgc1 (>= 1:7.4.2) but it is not installable
E: Unable to correct problems, you have held broken packages.
```

I tried it with chatGPT, tried everything it told to do, but still it throws the same error.

**Comment by ChefAharoni** [comment, marked as answer]
Hi @shikhar229169, please try the solution in this link: [Install make Ubuntu](https://linuxhint.com/install-use-make-ubuntu/).
It was mentioned here https://github.com/Cyfrin/foundry-full-course-f23/discussions/195.

**Reply by DevSwayam** [reply]
Are you a bot? or a real human.

**Reply by ChefAharoni** [reply]
A bot with special skills that has passed the [turing test](https://en.wikipedia.org/wiki/Turing_test).
<br><br>

Just kidding, I'm a real human being.

**Reply by shikhar229169** [reply]
@ChefAharoni, it still throws the same error while installing it.

**Reply by ChefAharoni** [reply]
Have you tried installing the required dependency?
Try installing it using [this guide](https://installati.one/ubuntu/21.10/libgc1/).

```bash
sudo apt update
```

```bash
sudo apt -y install libgc1
```

**Reply by DevSwayam** [reply]
Bro you literally know everything from this repo. Respect++

**Reply by shikhar229169** [reply]
@ChefAharoni, Yes I tried this already before making this thread, it throws the error:
```
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Package libgc1 is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'libgc1' has no installation candidate
```

**Reply by ChefAharoni** [reply]
Then seems like this particular package is giving you problems.
Have you tries [this fix](https://askubuntu.com/questions/391184/is-not-available-but-is-referred-to-by-another-package-finding-that-package)?

> You can search for the package with apt-cache:

```bash
apt-cache search x
```
> This will output all packages that in a way or another make a reference to x.

**Reply by shikhar229169** [reply]
![Screenshot (249)](https://github.com/Cyfrin/foundry-full-course-f23/assets/111875681/6f2f52ee-85b2-4989-9165-7202722a8029)
